EnglishFilter/sections/specific.txt

```diff
@@ -293,6 +293,7 @@ adwork.us,afly.pro##center > a[href*="?utm_source"]
 streamcr7.com##.container > a[href].btn-outline-primary
 sundryfiles.com##.download-timer-text + div[style="text-align:center;font-size:13px;color:#a1a1a1;"]
 news18.com##.pani-wid
+||content*.thousandbabes.com/images/thousandbabes.com/*.gif
```

Comment:
Hm, I have a different request `https://content2.thousandbabes.com/images/thousandbabes.com/banners/finestwomen.com/0.jpg`

Reply:
Also please block the leftover `thousandbabes.com##.sponsorholder`